tests/performance/specs/home-block-theme.test.js

```diff
@@ -40,14 +45,47 @@ describe( 'Server Timing - Twenty Twenty Three', () => {
 			const [ navigationTiming ] = JSON.parse( navigationTimingJson );
 
 			results.wpBeforeTemplate.push(
-				navigationTiming.serverTiming[0].duration
+				navigationTiming.serverTiming[ 0 ].duration
 			);
 			results.wpTemplate.push(
-				navigationTiming.serverTiming[1].duration
+				navigationTiming.serverTiming[ 1 ].duration
 			);
-			results.wpTotal.push(
-				navigationTiming.serverTiming[2].duration
+			results.wpTotal.push( navigationTiming.serverTiming[ 2 ].duration );
+
+			const ttfb = await page.evaluate(
```

Comment:
Can you add a comment here citing some sort of reference for what this implementation is based on?